server/services/message-service.js

```diff
@@ -2,25 +2,25 @@ const auth = require('@azure/ms-rest-nodeauth')
 const MessageReceiver = require('./messaging/message-receiver')
 const scheduleMessageAction = require('./schedule-message-action')
 const paymentMessageAction = require('./payment-message-action')
-const { isProd, paymentQueueConfig, scheduleQueueConfig } = require('../config')
+const { isProd, paymentQueue, scheduleQueue } = require('../config')
 
-process.on('SIGTERM', function () {
-  messageService.closeConnections()
+process.on('SIGTERM', async function () {
+  await messageService.closeConnections()
   process.exit(0)
 })
 
-process.on('SIGINT', function () {
-  messageService.closeConnections()
+process.on('SIGINT', async function () {
+  await messageService.closeConnections()
   process.exit(0)
 })
 
 class MessageService {
   constructor (credentials) {
     this.closeConnections = this.closeConnections.bind(this)
```

Comment:
Yeah, could well be. I was going a bit belt and braces when trying to identify unresolved promises. Looks like this might have escaped my attention. I'll get it sorted.